src/apprt/gtk/ui/1.2/config-errors-dialog.blp

```diff
@@ -2,27 +2,27 @@ using Gtk 4.0;
 using Adw 1;
 
 Adw.MessageDialog config_errors_dialog {
-    heading: _("Configuration Errors");
-    body: _("One or more configuration errors were found. Please review the errors below, and either reload your configuration or ignore these errors.");
+  heading: _("Configuration Errors");
```

Comment:
My editor somehow always reformats this with 4 spaces (I tried setting the indent style, sadly it didn't work.) Is there a way to specify this so that it remains consistent?

Reply:
Other than running `blueprint-compiler format` to reformat I don't know.